### PR TITLE
DR-3030: Render All Divisions page

### PR DESCRIPTION
**Pull request by 7emansell**
## Ticket:

- JIRA ticket [DR-3030](https://newyorkpubliclibrary.atlassian.net/browse/DR-3030)

## This PR does the following:

- Creates page component for All Divisions page (dummy data/ mainly scaffolding)
- Updates DS to `3.2.0`

### Checklist:

- [x] I have added relevant accessibility documentation for this pull request.
- [x] All new and existing tests passed.
- [x] I have updated the CHANGELOG.md.

[DR-3030]: https://newyorkpubliclibrary.atlassian.net/browse/DR-3030?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/pages/divisionsPage/divisionsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      activePage="divisions"
+      breadcrumbs={[
+        { text: "Home", url: "/" },
+        { text: "Divisions", url: "/divisions" },
+      ]}
+    >
+      <Box
+        sx={{
+          [`@media screen and (min-width: ${headerBreakpoints.smTablet})`]: {
+            maxWidth: "715px",
```
Comment (EdwinGuzman) [inline src/components/pages/divisionsPage/divisionsPage.tsx:31]: Is this the value from figma? Seems a bit random.

File: src/components/pages/divisionsPage/divisionsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          },
+          "> hgroup > p": {
+            fontWeight: "400 !important",
```
Comment (EdwinGuzman) [inline src/components/pages/divisionsPage/divisionsPage.tsx:34]: Ah, for the `subtitle`?

File: src/components/pages/divisionsPage/divisionsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+import { props } from "__tests__/data/swimlaneProps";
+import useBreakpoints from "src/hooks/useBreakpoints";
+
+export default function DivisionsPage() {
+  const { isLargerThanLargeTablet, isLargerThanLargeMobile } = useBreakpoints();
+  const numColumns = isLargerThanLargeTablet
+    ? 4
+    : isLargerThanLargeMobile
+    ? 2
+    : 1;
```
Comment (EdwinGuzman) [inline src/components/pages/divisionsPage/divisionsPage.tsx:19]: This is the same as what's in `HomePageMainContent`. I think now is a good time to make this into a custom hook to reuse it in both components, and maybe any other ones in the future.

File: src/components/pageLayout/pageLayout.tsx [hunk 4]
```diff
@@ -95,7 +96,16 @@ const PageLayout = ({
               breadcrumbsType="digitalCollections"
               breadcrumbsData={breadcrumbs || []}
             />
-            <TemplateAppContainer contentPrimary={children as JSX.Element} />
+            <Box
```
Comment (EdwinGuzman) [inline src/components/pageLayout/pageLayout.tsx:99]: I'd like the `TemplateAppContainer` component to be more flexible and allow you to pass in these values. Can you add a TODO to update this later?

I don't want to block the work but I want to move closer to a better `TemplateAppContainer`.

**Review by EdwinGuzman — CHANGES_REQUESTED** [review]
Some minor details to update. Can you also update the changelog?

File: src/components/pages/divisionsPage/divisionsPage.tsx (hunk 1 above)
Comment (7emansell) [reply src/components/pages/divisionsPage/divisionsPage.tsx:31]: Yup

File: src/components/pages/divisionsPage/divisionsPage.tsx (hunk 2 above)
Comment (7emansell) [reply src/components/pages/divisionsPage/divisionsPage.tsx:34]: Yes :(

File: CHANGELOG.md [hunk 5]
```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Updated
 
 - Updated static breadcrumbs to render correct copy (DR-3023)
+- Updated `/divisions` page to render title and content
```
Comment (EdwinGuzman) [inline CHANGELOG.md:18]: Add ticket number.

**Review by EdwinGuzman — APPROVED** [review]

**Review by avertrees — APPROVED** [review]

**Pull request merged**